scripts/sct_dmri_moco.py

```diff
@@ -40,78 +41,109 @@ def get_parser():
-                      mandatory=False,
-                      default_value=param_default.group_size,
-                      example=['2'])
-    parser.add_option(name='-m',
-                      type_value='file',
-                      description='Binary mask to limit voxels considered by the registration metric.',
-                      mandatory=False,
-                      example=['dmri_mask.nii.gz'])
-    parser.add_option(name='-param',
-                      type_value=[[','], 'str'],
-                      description="Advanced parameters. Assign value with \"=\"; Separate arguments with \",\"\n"
-                                  "poly [int]: Degree of polynomial function used for regularization along Z. For no regularization set to 0. Default=" + param_default.poly + ".\n"
-                                                "smooth [mm]: Smoothing kernel. Default=" + param_default.smooth + ".\n"
-                                                  "metric {MI, MeanSquares, CC}: Metric used for registration. Default=" + param_default.metric + ".\n"
-                                                  "gradStep [float]: Searching step used by registration algorithm. The higher the more deformation allowed. Default=" + param_default.gradStep + ".\n"
-                                                    "sample [None or 0-1]: Sampling rate used for registration metric. Default=" + param_default.sampling + ".\n",
-                      mandatory=False)
-    parser.add_option(name='-x',
-                      type_value='multiple_choice',
-                      description='Final Interpolation.',
-                      mandatory=False,
-                      default_value=param_default.interp,
-                      example=['nn', 'linear', 'spline'])
-    parser.add_option(name='-ofolder',
-                      type_value='folder_creation',
-                      description='Output folder',
-                      mandatory=False,
-                      default_value=param_default.path_out,
-                      example='dmri_moco_results/')
-    parser.usage.addSection('MISC')
-    parser.add_option(name="-r",
-                      type_value="multiple_choice",
-                      description='Remove temporary files.',
-                      mandatory=False,
-                      default_value='1',
-                      example=['0', '1'])
-    parser.add_option(name="-v",
-                      type_value='multiple_choice',
-                      description="verbose: 0 = nothing, 1 = classic, 2 = expended",
-                      mandatory=False,
-                      example=['0', '1', '2'],
-                      default_value='1')
+    parser = argparse.ArgumentParser(
+        description=(
+            '  Motion correction of dMRI data. Some of the features to improve robustness were proposed in Xu et al. '
+            '(http://dx.doi.org/10.1016/j.neuroimage.2012.11.014) and include:\n'
+            '    -group-wise (-g)\n'
```

Comment:
i think the 2-space indent is sufficient. Also, i think it is important to keep a space between "-" and the item, otherwise people might think this is a flag (whereas this is a list of items). Same comment applies to the other items below
```suggestion
            '  - group-wise (-g)\n'
```